contrib/sip_proxy/filters/network/source/router/router_impl.h

```diff
@@ -293,6 +294,13 @@ class Router : public Upstream::LoadBalancerContextBase,
   QueryStatus handleCustomizedAffinity(std::string type, std::string key,
                                        MessageMetadataSharedPtr metadata);
 
+  absl::string_view localAddress() {
+    auto& local_address =
+        context_.getTransportSocketFactoryContext().localInfo().address()->ip()->addressAsString();
+    ENVOY_LOG(debug, "localAddress {}", local_address);
```

Comment:
This code is repeated twice, how about putting it in a a utility?

Reply:
move into `utility.h`